Examples/restcomm-olympus/app/build.gradle

```diff
@@ -67,6 +67,7 @@ android {
             // Not specifying applicationIdSuffix since release uses the default applicationId: org.restcomm.android.olympus
             buildConfigField "boolean", "ENABLE_TEST_FAIRY_RUNTIME", "true"
             buildConfigField "String", "TESTFAIRY_APP_TOKEN", TESTFAIRY_APP_TOKEN
+            buildConfigField "String", "TEST_PUSH_FCM_KEY", TEST_PUSH_FCM_KEY
```

Comment:
I think we should also put this in debug build configuration so that we can use push and test with push on any build configuration.

Reply:
Whoops, missed it; cool